packages/run-protocol/src/vaultFactory/vaultDirector.js

```diff
@@ -143,6 +143,28 @@ const getCollaterals = async ({ state }) => {
   );
 };
 
+const getLiquidationConfig = directorParamManager => ({
+  install: directorParamManager.getInstallation(LIQUIDATION_INSTALL_KEY),
+  terms: directorParamManager.getUnknown(LIQUIDATION_TERMS_KEY),
```

Comment:
This should work as well, and type-check equivalently.  Your preference.
```suggestion
  install: directorParamManager.getLiquidationInstall(),
  terms: directorParamManager.getLiquidationTerms(),
```

Reply:
The getters are preferred as string args won't pass jessie-check.